app/src/main/java/com/github/se/wanderpals/MainActivity.kt

```diff
@@ -31,6 +87,34 @@ fun Greeting(name: String, modifier: Modifier = Modifier) {
   Text(text = "Hello $name!", modifier = modifier)
 }
 
+@Composable
+fun Google_button(onClick: () -> Unit) {
```

Comment:
Need Documentation for Google_button (there should be a documentation convention concerning Kotlin and composable)